app/Http/Controllers/PostController.php

```diff
@@ -3,10 +3,15 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
 
 class PostController extends Controller
 {
-   public function index(){
-       return view('posts.index');
+    public function index(){
+     $id = 5; 
+    $posts = DB::table ('posts')
+            ->orderBy('created_at', 'desc')
+            ->first();
+    dd($posts);
```

Comment:
```suggestion
        $posts = DB::table ('posts')
            ->orderBy('created_at', 'desc')
            ->first();
```
Indenting is niet juist en je hebt een dd in je code staan dat is niet juist.